### PR TITLE
Inspector general link and footer updates

### PR DESCRIPTION
**Pull request by dannychapman**
Completes #871 and reduces footer complexity. Please review @emilyville

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -104,7 +104,7 @@
   <div data-alert class="alert-box notice">
     <div class="row full">
       <div class="small-12 message">
-        <p><strong>This site is a <a href="https://www.vets.gov/playbook/design/">work in progress.</a></strong> If you don&rsquo;t find what you need, visit <a href="http://www.va.gov">VA.gov</a>.</p>
+        <p><span><strong>This site is a <a href="https://www.vets.gov/playbook/design/">work in progress.</a></strong> If you don&rsquo;t find what you need, visit <a href="http://www.va.gov">VA.gov</a>.</span></p>
```
Comment (emilyville) [inline _includes/header.html]: small nit - do we need this extra span tag? It looks empty.

**Comment by emilyville** [comment]
Sent you one (very very small) comment - other than that looks good!

**Comment by dannychapman** [comment]
@emilyville Removed extra span!

**Comment by emilyville** [comment]
:+1:  I will add corresponding ticket to add this to the gem + other footers.

**Pull request merged**